lib/build.spec.js

```diff
@@ -145,6 +149,30 @@ describe('build', () => {
         expect(babelLoaderRuleOptions.presets).toBeUndefined();
         expect(babelLoaderRuleOptions.plugins).toBeUndefined();
       });
+
+      it('should not alter the default babelOpts when a valid babel config is found in a monorepo', async () => {
+        const stats = await tempy.directory.task(async (directory) => {
+          await Promise.all([
+            writeFile(`${directory}/.git/HEAD`, ''),
+            writeFile(
+              `${directory}/packages/netlify-site/functions/index.js`,
+              'module.exports = () => console.log("hello world")',
+            ),
+            writeFile(`${directory}/babel.config.js`, ''),
+          ]);
+
+          const stats = await build.run(`packages/netlify-site/functions`, {
+            cwd: directory,
+          });
+          return stats;
```

Comment:
Nitpicking here, but I would straight up `return await build.run..`, the intermediate `stats` variable seems unnecarry and makes things more confusing (as it's shadowing the outer  `stats` variable)

Reply:
Good point, I'll fix it